homeassistant/components/device_tracker/asuswrt.py

```diff
@@ -16,7 +16,8 @@
 
 from homeassistant.components.device_tracker import (
     DOMAIN, PLATFORM_SCHEMA, DeviceScanner)
-from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.const import (CONF_HOST, CONF_PASSWORD,
+    CONF_USERNAME, CONF_PORT)
```

Comment:
Newline immediately after opening the bracket `(` and then 4 spaces indented on next line start with first one